modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -65,8 +65,8 @@ export interface SchemaElement extends ElementRule {
   pattern?: RegExp;
 }
 
-export interface SchemaMap { [name: string]: {} }
-export interface SchemaRegExpMap { [name: string]: RegExp }
+export interface SchemaMap {[name: string]: {}}
```

Comment:
These whitespace changes should have been done. In fact very soon eslint will fail with this change.